packages/py-moose-lib/moose_lib/main.py

```diff
@@ -10,6 +11,26 @@
 class StreamingFunction:
     run: Callable
 
+# class IngestionFormat(Enum):
+#     JSON = "JSON"
+#     JSON_ARRAY = "JSON_ARRAY"
+
+# T = TypeVar('T')
+
+# @dataclass
+# class IngestionConfig:
+#     format: Optional[IngestionFormat] = None
+
+# @dataclass
+# class StorageConfig(Generic[T]):
```

Comment:
In TS I was trying to be cute and used `keyof T` for `order_by_fields`. Python probably can't do this, so we don't need the generic type param.